libs/pika/async_cuda/tests/unit/cuda_sender.cu

```diff
@@ -126,8 +125,10 @@ int pika_main(pika::program_options::variables_map& vm)
     std::size_t device = vm["device"].as<std::size_t>();
     //
     unsigned int seed = (unsigned int) std::time(nullptr);
+#if !defined(PIKA_HAVE_HIP)
     if (vm.count("seed"))
         seed = vm["seed"].as<unsigned int>();
+#endif
 
     std::cout << "using seed: " << seed << std::endl;
```

Comment:
Please add a message here for the HIP case saying that the `--seed` command line option is ignored.